src/test/engineeffectsdelay_test.cpp

```diff
@@ -65,8 +65,10 @@ TEST_F(EngineEffectsDelayTest, NegativeDelayValue) {
 #endif
 }
 
+/*
 TEST_F(EngineEffectsDelayTest, DelayGreaterThanDelayBufferSize) {
-    const SINT numDelayFrames = mixxx::audio::SampleRate::kValueMax;
+    const SINT numDelayFrames =
+            mixxx::audio::SampleRate::kValueMax * mixxx::kEngineChannelCount + 2;
```

Comment:
great that you were able to figure out the reason the tests failed. I think if you fix this test, we can merge (theoretically, we still need to wait for 22.04).

Reply:
Okay, I just would like to fix the mentioned delay value issue in this PR too (clamping the delay to [0;kMaxDelay]).